### ISSUE TITLE
Thinner line for spectrum graph

### ISSUE THREAD

**Issue by sadko4u** [sadko4u/lsp-plugins#317]
Received one more feature request from the feedback form:

It would be nice to have a thinner line in the frequency graph and spectrum graph. Especially in the LSP Spectrum Analyzer (in analyzer and mastering mode) would be welcome if the moving line would be as thin as the flat horizontal line that goes up and down (or as the selector line).

Also in some other LSP plugins, it would be nice to have those specific lines in the spectrum- and timegraph thinner, or have an option to do so.

**Comment by CurlyMC** [comment]
Dear LSP-Team, best Vladimir,

Just some more **eyecandy** asking ;
Looking at the frequency line, I notice that the low end ( < 200Hz ) looks 'blocky', both in :
* the Spectrum Analyzer plugin ( when selected mode = analyzer ) 
* in the spectrum graph displays from EQ plugins ( both parametric- and graphic EQ's )

Don't know if this is intentional, and a specific choice made by the developers...
Although the mastering mode of the spectrum analyzer and the JaPa analyser are both nice and very usefull when mixing, 
maybe an option to implement the smooth mastering mode from the spectrum analyzer plugin into the LSP EQ ?

Or is it possible to increase the 'low end resolution' of the Spectrum Analyzer plugin max Tolerance value to 32768 or 65536 ( the FFT Frame value ?? ), without burning the CPU :-).

Kind Regards, Luc.

![Freq-Analyzer-Resolution](https://user-images.githubusercontent.com/72344251/230731640-cbebf51f-43e7-416e-95e5-92c971752d68.jpg)

**Comment by sadko4u** [comment]
The FFT transform means that we split the whole frequency range into the `N+1` frequencies starting with 0 and ending with the sample_rate/2 (including this one). The overall number of frequencies is defined by the `Tolerance` parameter. That means, if we specify `Tolerance=16384`, the number of frequencies `N` will be `8192`.
But the main problem of FFT is that frequencies have linear stepping:
```
f[i] = (i * sample_rate) / (2 * N)
```

That means, for tolerace 16384 and sample rate 48000 hZ we will get:
```
f[0] = 0
f[1] = 1 * 48000 / 16384 = 2.9296875 Hz
f[2] = 2 * 48000 / 16384 = 5.859375 Hz
f[3] = 8.7890625 Hz
f[4] = 11.71875 Hz
f[5] = 14.6484375 Hz
....

```

See? There is no continuous line that can be drawn in logarithmic scale. All you see in other applications is just an approximation to fill the gap between the magnitude values of the consequent frequencies.

LSP does not perform any approximation except the only `mastering` mode in the spectrum analyzer.

**Comment by CurlyMC** [comment]
Hi there Vladimir,

Thanx for the reply and explanation, now I do understand.
Happy to see that LSP-plugin's are telling the truth in logarithmic scale, and don't approximate :-)

**Comment by sadko4u** [comment]
Available in 1.2.7
